test/rubygems/utilities.rb

```diff
@@ -186,6 +186,41 @@ def self.create(body:, code:, msg:, headers: {})
   end
 end
 
+##
+# A fake Gem::FakeBrowser is used in tests to mock a browser in that it can
+# send HTTP requests to the defined URI.
+#
+# Example:
+#
+#   # Sends a get request to http://localhost:5678
+#   Gem::FakeBrowser.get URI("http://localhost:5678")
+#
+# See RubyGems' tests for more examples of FakeBrowser.
+#
+
+class Gem::FakeBrowser
```

Comment:
What about `TestBrowser` or `MockBrowser`?

Reply:
I took `FakeBrowser` from `FakeFetcher` above. I do think `MockBrowser` is a better name so I changed it to that 👍